frontend/src/components/worksheets/Worksheet.js

```diff
@@ -669,16 +672,18 @@ class Worksheet extends React.Component {
             },
         );
         
-        // Confirm operation
-        Mousetrap.bind(['enter'], function(e) {
-            if (this.state.openDelete){
-                this.executeBundleCommandNoEvent('rm');
-            }else if (this.state.openKill){
-                this.executeBundleCommandNoEvent('kill');
-            }else if (this.state.openDetach){
-                this.executeBundleCommandNoEvent('detach');
-            }
-        }.bind(this));
+        if (this.state.showBundleOperationButtons && (this.state.openDelete||this.state.openKill||this.state.openDelete)){
```

Comment:
It looks like we're checking `this.state.openDelete` twice.

Reply:
Good catch, thanks!